nvdaHelper/localWin10/oneCoreSpeech.h

```diff
@@ -37,22 +45,52 @@ class OcSpeech {
 	void setVolume(double volume);
 	double getRate();
 	void setRate(double rate);
+	void performCallback(
+		BYTE* data,
+		int length,
+		const wchar_t* markers
+	);
 };
 
+enum InstanceState {notInitialized, active, terminated};
+
+class InstanceManager {
+private:
+	static std::recursive_mutex _instanceStateMutex;

```

Comment:
This and SpeakThreadGuard don't need to be in this header. They aren't used outside of the oneCoreSpeech.cpp compilation unit, and don't need to have their compilation tied to oneCoreSpeech. We'll likely need them separate for OCR.